scripts/bfb-install

```diff
@@ -216,17 +216,13 @@ push_boot_stream_via_local_rshim()
     # Set SWINT2.BIT2 for DPU mode
     if [ ${nic_mode} -eq 0 ]; then
       ${BF_REG} $(basename ${rshim_node}) ${RSH_SWINT}.64 0x4 >/dev/null
+      echo ${BF_REG} $(basename ${rshim_node}) ${RSH_SWINT}.64 0x4
```

Comment:
debug message?
The debug messages should not be visible by default (to customer).